clusters/build-clusters/01_cluster/assets/99-master-ssh_machineconfig.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfig
+metadata:
+  labels:
+    machineconfiguration.openshift.io/role: master
+  name: 99-master-ssh
+spec:
```

Comment:
Can we remove all the empty sections to be sure this wont reset any config?

Reply:
I will try to apply it after the incident. I guess no need to create the noise right now.

Reply:
Updated.